src/app/queries/mocks/plans.mock.ts

```diff
@@ -161,40 +160,160 @@ const plan2: IPlan = {
 MOCK_PLANS = [plan1, plan2];
 
 export const migration1: IMigration = {
```

Comment:
Ah, I hadn't realized we already had `IMigration`. What's the reason for having both this and the `ITempMockMigration` from above?

Reply:
@mturley, this is temporary to avoid touching back and forth the structure until we have more visibility on those structure.
But that's annoying. Let me see if I can do better now that I have the UI in place.

Reply:
I've removed it after consolidating the mock migrations.